core/src/main/codegen/templates/Parser.jj

```diff
@@ -5157,6 +5157,33 @@ TimeUnit Second() :
     <SECONDS> { return warn(TimeUnit.SECOND); }
 }
 
+TimeUnit Millisecond() :
```

Comment:
This is duplicated both here and in the BodoSQl repo purely for testing, correct?

Reply:
Also, what exactly does warn do here? I'm not certain we want to be throwing warnings, because for SF users this might be totally normal.

Reply:
Ok, scratch that, it only throws a warning if compliance is set to strict, I think. so, this LGTM